plugin/src/main/java/org/opensearch/ml/action/models/UpdateModelTransportAction.java

```diff
@@ -492,7 +492,7 @@ private ActionListener<UpdateResponse> getUpdateResponseListenerWithUpdateModelC
                         String[] nodeIds = getUpdateModelCacheFailedNodesList(modelId, r);
                         log
                             .error(
-                                "Successfully update ML model index with model ID {} but update model cache was failed on following nodes {}",
+                                "Successfully update ML model index with model ID {} but update model cache was failed on following nodes {}, maybe retry?",
```

Comment:
> maybe retry?
Let's give the clear suggestion rather than asking user, change to `please retry`?

Reply:
Ack, will add